spec/tapioca/dsl/compilers/rubocop_spec.rb

```diff
@@ -0,0 +1,149 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "spec_helper"
+require "rubocop"
+require "rubocop-sorbet"
+
+module Tapioca
+  module Dsl
+    module Compilers
+      class RuboCopSpec < ::DslSpec
+        EXISTING_CONSTANTS = Runtime::Reflection
+          .descendants_of(::RuboCop::Cop::Base)
+          .filter_map { |constant| Runtime::Reflection.name_of(constant) }
```

Comment:
So as to be able to focus on just the constants we introduce in our tests, we need to compile a list of all the constants we'll gather from gems **before we define a bunch in our tests**.